src/processor_arm.cpp

```diff
@@ -10,10 +10,16 @@
 #include <fstream>
 #include <algorithm>
 
-#if defined(_CPU_AARCH64_) || __GLIBC_PREREQ(2, 16)
+// This nesting is required to allow compilation on musl
+#define DYN_GETAUXVAL
```

Comment:
Can you change this name to `USE_DYN_GETAUXVAL`, so it's clear it's our variable and not affecting the headers? Otherwise, LGTM.

Reply:
I'm just moving around the preprocessor statements, not adding new variables. I'm not actually sure what that variable does :smile:

Reply:
It just defines later for us whether we've got the sys/auxv header, so that we can link against `getauxval` directly, or should try to get it dynamically.

Reply:
Sorry about the delay! The change has been made :slightly_smiling_face: